ui-admin/package.json

```diff
@@ -57,5 +57,11 @@
   },
   "eslintConfig": {
     "extends": "../.eslintrc.js"
+  },
+  "jest": {
+    "moduleNameMapper": {
+      "react-markdown": "<rootDir>/../node_modules/react-markdown/react-markdown.min.js",
```

Comment:
This seems unnecessary, since it's resolving react-markdown from node_modules as it would be normally.

Reply:
I couldn't get tests to pass without it.  They fail with
```
Jest encountered an unexpected token

    Jest failed to parse a file. This happens e.g. when your code or its dependencies use non-standard JavaScript syntax, or when Jest is not configured to support such syntax.

/Users/dbush/code/pearl/node_modules/react-markdown/index.js:6
    export {uriTransformer} from './lib/uri-transformer.js'
    ^^^^^^

    SyntaxError: Unexpected token 'export'

       8 | var _classnames = _interopRequireDefault(require("classnames"));
       9 | var _react = _interopRequireDefault(require("react"));
    > 10 | var _reactMarkdown = _interopRequireDefault(require("react-markdown"));
```